examples/gradient_descent.py

```diff
@@ -23,12 +22,14 @@ def gradient_descent(start,
                      lr=0.05,
                      max_iter=128,
                      precision=1e-5):
+    """Operate a gradient descent on a given manifold until either max_iter or
+    a given precision is reached."""
     x = start
     for i in range(max_iter):
         x_prev = x
-        euclidian = - lr * grad(x)
-        tangent_vec = manifold.projection_to_tangent_space(vector=euclidian,
-                                                           base_point=x)
+        euclidian_grad = - lr * grad(x)
```

Comment:
people tend to use "euclidean" instead of "euclidian" https://en.wikipedia.org/wiki/Euclidean